### DISCUSSION TITLE
Show the version of Jellyfin Media Player inside the application

### DISCUSSION THREAD

**Discussion by djalan** [jellyfin/jellyfin-desktop#403] [Ideas]
Hello

It might just be me but I cannot find the version of the player when it is opened.

This might be a good place to put it:

![image](https://user-images.githubusercontent.com/707389/225482650-f5547aa6-ae67-4373-9f74-659f425faa9d.png)

**Comment by iwalton3** [comment]
On non-windows platforms you can do `jellyfinmediaplayer --version`. It should also show in the admin dashboard and where the application is installed. I don't think I could add it under the main menu but I _could_ put it in client settings.

**Comment by JonasR** [comment]
This would indeed be nice to have. I was also looking for it today and eventually realized its displayed in Windows' Apps & features menu.
<img width="613" alt="image" src="https://github.com/jellyfin/jellyfin-media-player/assets/307785/5af37d44-3e5e-4c93-b811-05724bf2118a">
